chore(doc): added section on copy command to "Importing data in bulk … (#967)
questdb/questdb.io: docs/reference/configuration.md

| cairo.sql.column.purge.retry.limit.days        | 31                | Number of days purge system will continue to re-try deleting stale column files before giving up.                                                                                                                        |
| cairo.system.table.prefix                      | sys.              | Prefix of the tables used for QuestDB internal data storage. These tables are hidden from QuestDB web console.                                                                                                           |

### Bulk CSV import

This section describes configuration settings for using `COPY` to import large CSV files.

| Property                            | Default | Description                                                                                                                                                                                           |
|-------------------------------------|---------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|

> | cairo.iouring.enabled               | true    | Enable or disable io_uring implementation . Can be used to switch it off if there's a kernel bug affecting it.                                                                                        |

Reviewer: ```suggestion
| cairo.iouring.enabled               | true    | Enable or disable io_uring implementation. Applicable to newer Linux kernels only. Can be used to switch io_uring interface usage off if there's a kernel bug affecting it.                                                                                        |
```

| cairo.sql.copy.buffer.size          | 2 MiB   | Size of read buffers used in import                                                                                                                                                                   |
| cairo.sql.copy.log.retention.days   | 3       | Number of days to keep import messages in sys.text_import_log .                                                                                                                                       |

> | cairo.sql.copy.max.index.chunk.size | 100MiB  | Maximum size of index chunk file used to limit total memory requirements of import . Indexing phase should use roughly threads_count * cairo.sql.copy.max.index.chunk.size  of memory.               |

Reviewer: ```suggestion
| cairo.sql.copy.max.index.chunk.size | 100m  | Maximum size of index chunk file used to limit total memory requirements of import. Indexing phase should use roughly thread_count * cairo.sql.copy.max.index.chunk.size of memory.               |
```

> | cairo.sql.copy.queue.capacity       | 32      | Size of copy task queue. Should be increased if there's more than 32 import workers .                                                                                                                |

Reviewer: ```suggestion
| cairo.sql.copy.queue.capacity       | 32      | Size of copy task queue. Should be increased if there's more than 32 import workers.                                                                                                                |
```

| cairo.sql.copy.root                 | null    | Input root directory for CSV imports via `COPY` SQL. This path should not overlap with other directory (e.g. db, conf) of running instance, otherwise import may delete or overwrite existing files. |

> | cairo.sql.copy.work.root            | null    | Temporary import file directory . Same as `cairo.sql.copy.root` if not set explicitly.                                                                                                                |

Reviewer: ```suggestion
| cairo.sql.copy.work.root            | null    | Temporary import file directory. Same as `cairo.sql.copy.root` if not set explicitly.                                                                                                                |
```

| cairo.system.table.prefix                      | sys.              | Prefix of the tables used for QuestDB internal data storage. These tables are hidden from QuestDB web console.                                                                                                           |

### CSV import

This section describes configuration settings for using `COPY` to import large CSV files.

Mandatory settings to enable `COPY`:

| Property                            | Default | Description                                                                                                                                                                                           |
|-------------------------------------|---------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
| cairo.sql.copy.root                 | null    | Input root directory for CSV imports via `COPY` SQL. This path should not overlap with other directory (e.g. db, conf) of running instance, otherwise import may delete or overwrite existing files. |

> | cairo.sql.copy.work.root            | null    | Temporary import file directory. Same as `cairo.sql.copy.root` if not set explicitly.                                                                                                                |

Reviewer 2: @amunra yes, paths are validated thoroughly for jailbreaks and overlaps with other paths

|-------------------------------------|---------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
| cairo.iouring.enabled               | true    | Enable or disable io_uring implementation. Applicable to newer Linux kernels only. Can be used to switch io_uring interface usage off if there's a kernel bug affecting it.                                                                                        |
| cairo.sql.copy.buffer.size          | 2 MiB   | Size of read buffers used in import.                                                                                                                                                                   |
| cairo.sql.copy.log.retention.days   | 3       | Number of days to keep import messages in `sys.text_import_log`.                                                                                                                                       |

> | cairo.sql.copy.max.index.chunk.size | 100MiB  | Maximum size of index chunk file used to limit total memory requirements of import . Indexing phase should use roughly threads_count * cairo.sql.copy.max.index.chunk.size  of memory.               |

Reviewer: ```suggestion
| cairo.sql.copy.max.index.chunk.size | 100m  | Maximum size of index chunk file used to limit total memory requirements of import. Indexing phase should use roughly `thread_count * cairo.sql.copy.max.index.chunk.size` of memory.               |
```
